pkgs/by-name/qu/quarkus/package.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, jre
```

```suggestion
, jdk
```
https://github.com/NixOS/nixpkgs/blob/a9a3ea33b666758058d001edcc809b9a2bb3ff44/pkgs/top-level/all-packages.nix#L16299-L16306

---

Is it a good idea ? Semantically, using explicitly jre has a strong meaning imo.  
Otherwise, Can the use of jre_minimal be something ?

---

What are you thought on this ? @jlesquembre

---

I still prefer `jdk`, since Java 9, Java no longer ships a separate JRE package, and `jre` is just an alias.

About `jre_minimal`, my opinion is that it makes a lot of sense when you deploy your application, build a container, etc, but not as a dependency of a derivation on nixpkgs.

---

I see your point. I have update the MR.